recipes/c-client/all/patches/0001-fix-yunchan-tempfile.patch

```diff
@@ -0,0 +1,68 @@
+--- src/osdep/nt/yunchan.c
++++ src/osdep/nt/yunchan.c
+@@ -219,6 +219,34 @@
+  * I would be delighted to have a better alternative.
+  */
+ 
++/* Fixes the above regret :-) */
++static FILE *win_create_tempfile (char *name)
++{
++  HANDLE handle = INVALID_HANDLE_VALUE;
++  int descriptor = -1;
++  FILE *result = NIL;
```

Comment:
what is `NIL`? shouldn't it be `NULL` instead?

Reply:
Yes, `NIL` and `T` ;)

Reply:
okay, perhaps, author liked pascal xD

Reply:
I got Scheme impressions personally, I believe that was a common language at universities at the time :)